### DISCUSSION TITLE
InfluxDB. . . . again

### DISCUSSION THREAD

**Discussion by TLatGH** [jomjol/AI-on-the-edge-device#1258] [Q&A]
Ich probiere es mal in Deutsch

Anscheinend wird hier der Zugang zu InfluxDB Cloud verwendet, da im Code die Rede von Bucket ist?

gibt es die Möglichkeit den Zugang mit den Daten auch für die "alte" Version von InfluxDB anzupassen.
Meine aktuellen Sensoren senden alle an Corlysis die anscheinend noch auf die alte Version von InfluxDB setzen.

**Comment by caco3** [comment]
The term `bucket` is not only used in the cloud, InfluxDB uses it s a general term.
I am confident that it is mainly designed to work locally.

Also note that it only works with InfluxDB1 but not with InfluxDB2!

**Comment by jomjol** [comment]
@caco3 hat vollkommen recht. Die implementierte Version InfluxDB-Version funktioniert nur bis 1.8. Ab 2.0 hat sich das Interface geändert und das wird aktuell nicht unterstützt.
